loopback/server/server.js

```diff
@@ -1,8 +1,15 @@
 var loopback = require('loopback');
 var boot = require('loopback-boot');
+var helmet = require('helmet');
 
 var app = module.exports = loopback();
 
+app.use(helmet.ieNoOpen())
+app.use(helmet.noCache())
```

Comment:
What benefit is `noCache` ?
I think it causes performance issue.

Reply:
ah, you're right.
I'll removed that.